app/routes/_gcn.circulars._archive._index/DateSelectorMenu.tsx

```diff
@@ -0,0 +1,162 @@
+/*!
+ * Copyright © 2023 United States Government as represented by the
+ * Administrator of the National Aeronautics and Space Administration.
+ * All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { useSearchParams, useSubmit } from '@remix-run/react'
+import {
+  Button,
+  CardBody,
+  CardFooter,
+  DateRangePicker,
+  Grid,
+  Icon,
+  Radio,
+} from '@trussworks/react-uswds'
+import { useState } from 'react'
+
+import DateSelectorButton from './DateSelectorButton'
+import DetailsDropdownContent from '~/components/DetailsDropdownContent'
+
+export default function ({
```

Comment:
We tend not to have default exports in library files.

```suggestion
export function DateSelector ({
```

Reply:
Ah, I was taking a cue from the way the DateSelectorButton file was written